lib/slack/web/config.rb

```diff
@@ -21,8 +22,8 @@ module Config
       def reset
         self.endpoint = 'https://slack.com/api/'
         self.user_agent = "Slack Ruby Client/#{Slack::VERSION}"
-        self.ca_path = `openssl version -a | grep OPENSSLDIR | awk '{print $2}'|sed -e 's/\"//g'`
-        self.ca_file = "#{ca_path}/ca-certificates.crt"
+        self.ca_path = OpenSSL::X509::DEFAULT_CERT_DIR
```

Comment:
If `OpenSSL` is not defined (guess we don't want hard dependency) then this would crash. So this can be done as

```ruby
self.ca_path = OpenSSL::X509::DEFAULT_CERT_DIR if defined?(OpenSSL)
```